.access.yml

```
@@ -56,6 +56,13 @@ policies:
       - write
     pull_request: true
     users_from_json: https://raw.githubusercontent.com/Quansight/open-gpu-server/main/access/conda-forge-users.json
+  - id: tensorflow-feedstock-osx-m4-policy
+    repo: tensorflow-feedstock
+    roles:
+      - admin
+      - maintain
+      - write
+    pull_request: true
```

We need a list of users here, right? Below's a proposal based on the intersection of the tensorflow maintainers and core (but no strong feelings about who exactly is on the list -- if it helps debugging, please add yourself @aktech and/or also @jaimergp, @isuruf)
```suggestion
    pull_request: true
    users:
      - h-vetinari
      - hmaarrfk
      - wolfv
      - xhochy
```

---

I copied from here: https://github.com/conda-forge/.cirun/blob/116b795e96f47879761c7a6a1c798c19d41d097d/.access.yml#L52-L57

Aren't these users already one of (?):

- admin
- maintain
- write

---

Perhaps it's just my misunderstanding; I commented on this because the other config groups either had `users:` or `users_from_json`.